### PR TITLE
chore: UnoEthereumBlockChain Sample - Update to net6

### PR DESCRIPTION
**Pull request by aesteves900**

### CODE REVIEW

File: UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml [hunk 1]
```diff
@@ -17,31 +18,17 @@
...
-        <StackPanel.Background>
-            <media:AcrylicBrush TintOpacity="0.7"
-                                                TintLuminosityOpacity="0.9"
-                                                TintColor="{ThemeResource SystemAccentColor}"
-                                                FallbackColor="{ThemeResource SystemAccentColor}"
-                                                BackgroundSource="Backdrop"
-                             xmlns:media="using:Windows.UI.Xaml.Media" />
-        </StackPanel.Background>
+                Margin="0"
+                BorderBrush="{StaticResource Backdrop}">
```
Comment (Xiaoy312) [inline UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml:22]: the background brush has been dropped, and there is an addition of foreground=black on the button later down
any justification on why we are doing that in a pure template update?

Comment (aesteves900) [reply UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml]: I did a mistake and remove the code accidentally. The code was restored and the background was bound. with the centralized brush.

Comment (Xiaoy312) [reply UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml]: btw, there are two different brushes: 
- one with tint=0.1 for border; and 
- one tint=0.7 for background

**Review by Xiaoy312 — APPROVED** [review]
otherwise, all good

File: UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml (hunk 1 above)
Comment (aesteves900) [reply UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml]: Thanks, Done!

**Pull request merged**